src/main/java/http/Cookie.java

```diff
@@ -0,0 +1,48 @@
+package http;
+
+import com.google.common.collect.Maps;
+import http.Const.HttpConst;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Cookie {
+    private Map<String, String> cookieMap = Maps.newHashMap();
```

Comment:
변수 초기화는 필드나 생성자 한 군데서만 해도 됩니다 :)
```suggestion
    private Map<String, String> cookieMap;
```